### PR TITLE
start Prior features and existing proposals section

### PR DESCRIPTION
**Pull request by tantek**
start Prior features and existing proposals section with existing HTML elements. left in a "..." to imply more coming in this section.

### CODE REVIEW

**Review by martinthomson — APPROVED** [review]
Gee, I'm not a fan of how `<data>` contains redundancy.  But I guess that you need that.

File: amount.md [hunk 1]
```diff
@@ -19,3 +19,21 @@ In particular, we should look for and prefer [declarative approaches](https://ww
...
+
+HTML also has the [`<data>` element](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-data-element) 
+which can be used to represent numerical amounts independent of punctuation, e.g.:
+
+`<data value=1000 lang=en>1,000</data>`
+`<data value=1.5 lang=fr>1,5</data>`
+
+No known examples of such usage in the wild however.
+
+Both are widely natively supported by browsers ([MDN data support](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/data#browser_compatibility), [MDN time support](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#browser_compatibility)).
```
Comment (zcorpan) [inline amount.md:37]: ...although they don't do anything interesting for those elements (nothing to solve the user problems described above).

File: amount.md [hunk 2]
```diff
@@ -19,3 +19,21 @@ In particular, we should look for and prefer [declarative approaches](https://ww
...
+and in particular [durations](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-duration-string) 
+amounts with units of seconds, minutes, hours, or days.
+
+HTML also has the [`<data>` element](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-data-element) 
+which can be used to represent numerical amounts independent of punctuation, e.g.:
+
+`<data value=1000 lang=en>1,000</data>`
+`<data value=1.5 lang=fr>1,5</data>`
+
+No known examples of such usage in the wild however.
```
Comment (zcorpan) [inline amount.md:35]: `time`: https://chromestatus.com/metrics/feature/timeline/popularity/1100
`data`: https://chromestatus.com/metrics/feature/timeline/popularity/1099

For `data`, looking at the first 11 pages, some don't have the element (anymore?) or use it incorrectly (no `value` attribute, no content, or using invalid attributes like `href`), or should have used `time`.

**Comment by tantek** [comment]
@martinthomson wrote:
> Gee, I'm not a fan of how `<data>` contains redundancy. But I guess that you need that.

Yup! And I'll be calling this out in the next section as a deficiency we can improve upon with a new element with a more specific processing model.

File: amount.md (hunk 1 above)
Comment (tantek) [reply amount.md]: > ...although they don't do anything interesting for those elements (nothing to solve the user problems described above).

Good caveat to point out inline. Edited PR to add a bit of text accordingly, including screen readers, with citation.

File: amount.md (hunk 2 above)
Comment (tantek) [reply amount.md]: > For `data`, looking at the first 11 pages, some don't have the element (anymore?) or use it incorrectly (no `value` attribute, no content, or using invalid attributes like `href`), or should have used `time`.

I meant specifically the kind of usage I was demonstrating with the examples, however that indicates my "No known examples" sentence was ambiguous so I've rewritten/expanded it accordingly with more precision.

**Review by zcorpan — APPROVED** [review]

**Pull request merged**